packages/common/package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "@babel/plugin-transform-flow-strip-types": "^7.0.0",
     "@babel/polyfill": "^7.4.4",
+    "@codesandbox/components": "^0.0.2",
```

Comment:
Is this still needed? Not sure if common is still using components.

Reply:
It does use it, we needed to keep it for preferences stuff